coredns/metadata.csv

```diff
@@ -1,83 +1,120 @@
+coredns.request_size.bytes.count,count,,byte,,[OpenMetrics V1 and V2] size of the request in bytes,0,coredns,request size
+coredns.request_size.bytes.bucket,count,,byte,,[OpenMetrics V2] sample size of the request in bytes,0,coredns,request size
+coredns.response_size.bytes.sum,count,,byte,,[OpenMetrics V1 and V2] size of the request in bytes,0,coredns,response size
+coredns.response_size.bytes.count,count,,byte,,[OpenMetrics V1 and V2] size of the request in bytes,0,coredns,response size
+coredns.response_size.bytes.bucket,count,,byte,,[OpenMetrics V2] sample size of the request in bytes,0,coredns,response size
+coredns.cache_size.count,gauge,,entry,,[OpenMetrics V1 and V2],1,coredns,cache size
+coredns.panic_count.count,count,,entry,,[OpenMetrics V1 and V2],1,coredns,total number of panics
+coredns.go.gc_duration_seconds.count,count,,second,,[OpenMetrics V1 and V2] Count of the GC invocation durations.,0,coredns,
+coredns.go.gc_duration_seconds.sum,count,,second,,[OpenMetrics V1 and V2] Sum of the GC invocation durations.,0,coredns,
+coredns.go.gc_duration_seconds.quantile,gauge,,second,,[OpenMetrics V1 and V2] Quantiles of the GC invocation durations.,0,coredns,
+coredns.go.goroutines,gauge,,thread,,[OpenMetrics V1 and V2] Number of goroutines that currently exist.,0,coredns,
+coredns.go.info,gauge,,,,[OpenMetrics V1 and V2] Information about the Go environment.,0,coredns,
+coredns.go.memstats.alloc_bytes,gauge,,byte,,[OpenMetrics V1 and V2] Number of bytes allocated and still in use.,0,coredns,
+coredns.go.memstats.alloc_bytes_total,count,,byte,,[OpenMetrics V1] Total number of bytes allocated even if freed.,0,coredns,
+coredns.go.memstats.buck_hash_sys_bytes,gauge,,byte,,[OpenMetrics V1 and V2] Number of bytes used by the profiling bucket hash table.,0,coredns,bytes used by profiling
+coredns.go.memstats.frees_total,count,,,,[OpenMetrics V1] Total number of frees.,0,coredns,
+coredns.go.memstats.frees_total.count,count,,,,[OpenMetrics V2] Total number of frees.,0,coredns,
+coredns.go.memstats.gc_cpu_fraction,gauge,,percent,,[OpenMetrics V1 and V2] CPU taken up by GC,0,coredns,CPU taken up by gc
+coredns.go.memstats.gc_sys_bytes,gauge,,byte,,[OpenMetrics V1 and V2] Number of bytes used for garbage collection system metadata.,0,coredns,
+coredns.go.memstats.heap_alloc_bytes,gauge,,byte,,[OpenMetrics V1 and V2] Bytes allocated to the heap,0,coredns,
+coredns.go.memstats.heap_idle_bytes,gauge,,byte,,[OpenMetrics V1 and V2] Number of idle bytes in the heap,0,coredns,
+coredns.go.memstats.heap_inuse_bytes,gauge,,byte,,[OpenMetrics V1 and V2] Number of Bytes in the heap,0,coredns,
+coredns.go.memstats.heap_objects,gauge,,object,,[OpenMetrics V1 and V2] Number of objects in the heap,0,coredns,
+coredns.go.memstats.heap_released_bytes,gauge,,byte,,[OpenMetrics V1 and V2] Number of bytes released to the system in the last gc,0,coredns,
```

Comment:
Is this metric also in V2? It seems very similar to `coredns.go.memstats.heap_released_bytes.count`

Reply:
Yeah, `coredns.go.memstats.heap_released_bytes` is a gauge metric that is present in both V1 and V2. Coredns v1.2.0 also has this metric as both a gauge and counter in omV2, `coredns.go.memstats.heap_released_bytes.count`.